airbyte-integrations/connectors/source-google-search-console/source_google_search_console/source.py

```diff
@@ -62,8 +63,21 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             SearchAnalyticsAllFields(**stream_config),
         ]
 
+        streams = streams + self.get_custom_reports(config=config, stream_config=stream_config)
+
         return streams
 
+    def get_custom_reports(self, config: Mapping[str, Any], stream_config: Mapping[str, Any]) -> List[Optional[Stream]]:
+        if "custom_reports" not in config:
+            return []
+
+        reports = json.loads(config["custom_reports"])
```

Comment:
For safety would you mind checking the `custom_reports` field is a list?

Reply:
Done by defining jsonschema